pom.xml

```diff
@@ -108,6 +110,16 @@
         <artifactId>grpc-google-cloud-bigquerystorage-v1beta2</artifactId>
         <version>0.85.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-bigquerystorage-v1beta2:current} -->
       </dependency>
+      <dependency>
```

Comment:
I don't think you need to add these two. Just move avro to the end of your dependencies section.

Reply:
will do

Reply:
Or maybe not. The avro dependency on jackson is not as deep.